llvm/test/CodeGen/AMDGPU/writelane_m0.mir

```diff
@@ -0,0 +1,14 @@
+# RUN: llc -mtriple=amdgcn-amd-amdhsa -mcpu=gfx904 -verify-machineinstrs -run-pass=legalizer -o - %s | FileCheck %s
```

Comment:
These tests should go in test/MachineVerifier, and only need -run-pass=none (don't even really need -verify-machineinstrs)